test/controllers/APIControllerSpec.scala

```diff
@@ -62,14 +62,29 @@ class APIControllerSpec extends SpecBase {
     }
 
     s"return OK (${Status.OK}) when the call returns some data and can be parsed to the correct response" in new Setup {
+      when(mockETMPService.getNumberOfEstimatedPenalties(mockETMPPayloadForAPIResponseData)).thenReturn(2)
```

Comment:
Oh and `ArgumentMatchers.any()` here
```suggestion
      when(mockETMPService.getNumberOfEstimatedPenalties(ArgumentMatchers.any())).thenReturn(2)
```

Reply:
Done.